### PR TITLE
Docs restyle [draft]

### PR DESCRIPTION
**Pull request by Kevingislason**

### CODE REVIEW

File: src/theme/NavbarItem/DropdownNavbarItem.js [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+                }}
+            >
+                {props.children ?? props.label}
+            </NavbarNavLink>
+            <ul className="dropdown__menu">
+                {items.map((childItemProps, i) => (
+                    <NavbarItem
+                        isDropdownItem
+                        onKeyDown={(e) => {
+                            if (i === items.length - 1 && e.key === "Tab") {
```
Comment (zhangjing28) [inline src/theme/NavbarItem/DropdownNavbarItem.js:93]: I think it's better extract a function to handle this event.

File: src/theme/NavbarItem/DropdownNavbarItem.js [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+    const containsActive = containsActiveItems(items, localPathname);
+    const { collapsed, toggleCollapsed, setCollapsed } = useCollapsible({
+        initialState: () => !containsActive,
+    });
+    // Expand/collapse if any item active after a navigation
+    useEffect(() => {
+        if (containsActive) {
+            setCollapsed(!containsActive);
+        }
+    }, [localPathname, containsActive, setCollapsed]);
```
Comment (zhangjing28) [inline src/theme/NavbarItem/DropdownNavbarItem.js:136]: I think `localPathname` is useless in here

File: src/components/Feedback/Bad.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import React from "react";
+export default function Bad({size=16,fill='black',...props}) {
```
Comment (zhangjing28) [inline src/components/Feedback/Bad.js:2]: The `Bad`, `Good`, `Normal` ... files have the different space with other files. They should be the same as the others

File: src/components/Feedback/styles.module.css [hunk 4]
```diff
@@ -0,0 +1,35 @@
+.content {
```
Comment (zhangjing28) [inline src/components/Feedback/styles.module.css:1]: Same as above.

File: src/theme/DocSidebarItem/Category/index.js [hunk 5]
```diff
@@ -0,0 +1,179 @@
...
+          'menu__list-item-collapsible--active': isCurrentPage,
+        })}>
+        <Link
+          className={clsx('menu__link', {
+            'menu__link--sublist': collapsible,
+            'menu__link--sublist-caret': !href && collapsible,
+            'menu__link--active': isActive,
+          })}
+          onClick={
+            collapsible
```
Comment (zhangjing28) [inline src/theme/DocSidebarItem/Category/index.js:137]: extract a function for this event

File: src/theme/Logo/index.js [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+import { useThemeConfig } from "@docusaurus/theme-common";
+import useBaseUrl from "@docusaurus/useBaseUrl";
+import useDocusaurusContext from "@docusaurus/useDocusaurusContext";
+import ThemedImage from "@theme/ThemedImage";
+import clsx from "clsx";
+import React from "react";
+import styles from "./styles.module.css";
+
+function LogoThemedImage({ logo, alt, imageClassName }) {
+    const sources = {
```
Comment (zhangjing28) [inline src/theme/Logo/index.js:11]: Pay attention to the space in this file

File: src/theme/NavbarItem/DefaultNavbarItem.js [hunk 7]
```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import clsx from "clsx";
+import NavbarNavLink from "@theme/NavbarItem/NavbarNavLink";
+function DefaultNavbarItemDesktop({
+    className,
+    isDropdownItem = false,
+    ...props
+}) {
+    const element = (
+        <NavbarNavLink
```
Comment (zhangjing28) [inline src/theme/NavbarItem/DefaultNavbarItem.js:10]: Same as above

File: src/theme/NavbarItem/DropdownNavbarItem.js [hunk 8]
```diff
@@ -0,0 +1,180 @@
...
+    Collapsible,
+} from "@docusaurus/theme-common";
+import {
+    isSamePath,
+    useLocalPathname,
+} from "@docusaurus/theme-common/internal";
+import NavbarNavLink from "@theme/NavbarItem/NavbarNavLink";
+import NavbarItem from "@theme/NavbarItem";
+function isItemActive(item, localPathname) {
+    if (isSamePath(item.to, localPathname)) {
```
Comment (zhangjing28) [inline src/theme/NavbarItem/DropdownNavbarItem.js:15]: Same as above

**Review by zhangjing28 — COMMENTED** [review]
Nice work! However this is a huge PR, pls try to narrow it down next time.

File: src/theme/NavbarItem/DropdownNavbarItem.js (hunk 2 above)
Comment (jincan39) [reply src/theme/NavbarItem/DropdownNavbarItem.js:136]: on the mobile and after localPathname changed, the active item needs to be updated

**Pull request closed without merging**